### PR TITLE
Minor change to .eslintrc.js

### PR DESCRIPTION
**Pull request by olafur-palsson**
I noticed a couple of inconsistencies in style. Majority of code passes lint with this option added.

### CODE REVIEW

**Comment by nukeop** [comment]
I think we've been using spaces inside curly braces in many places, especially in props. I don't remember many cases of the object shorthand syntax though. Does this cause many changes if you run eslint with --fix?

**Comment by nukeop** [comment]
Ok, I can agree with those changes. Are you still working on something more or can I merge this?

**Comment by olafur-palsson** [comment]
It's done now. Can merge. :)

File: app/actions/index.js [hunk 1]
```diff
@@ -41,9 +41,7 @@ export function sourcesSearch (terms, plugins) {
     Object.assign(searchResults, plugins.musicSources[i].search(terms));
   }
 
-  return {
-    type: SOURCES_SEARCH
-  };
+  return { type: undefined };
```
Comment (nukeop) [inline app/actions/index.js]: I think this will introduce a bug.

File: app/actions/toasts.js [hunk 2]
```diff
@@ -48,6 +49,7 @@ function generateNotification(title, details, icon, type, settings) {
     },
     type)));
 
+    // BUG: _ is undefined
```
Comment (nukeop) [inline app/actions/toasts.js]: Lodash inserts its _ into global scope.

File: app/actions/toasts.js [hunk 3]
```diff
@@ -37,6 +37,7 @@ export function info(title, details, icon, settings) {
   return generateNotification(title, details, icon, {info: true}, settings);
 }
 
+/** @deprecated */
```
Comment (nukeop) [inline app/actions/toasts.js]: This shouldn't be deprecated...

File: app/actions/index.js (hunk 1 above)
Comment (nukeop) [reply app/actions/index.js]: Nvm, actually this function can probably be deleted altogether.

Comment (olafur-palsson) [reply app/actions/index.js]: Yeah, I wasn't sure what it was going to be user for so I just exposed the bug.

**Comment by olafur-palsson** [comment]
This one is ready.

**Pull request merged**